README.md

```diff
@@ -78,6 +77,23 @@ One downside is that you will be foregoing sherlock's load
 balancing since you need to be connecting to the same login machine at each
 step.
 
+### SSH Port Forwarding Considerations
+
+Depending on your cluster, you will need to identify whether the compute nodes (not the login nodes) are isolated from the outside world or not (i.e can be ssh'd into directly). For Sherlock, they are isolated. For FarmShare they are not. This is important when we are setting up the ssh command to port forward from the local machine to the compute node. 
+
+For HPC's where the compute node is isolated from the outside world - like in sherlock. The ssh command basically establishes a tunnel to the login node, and then from the login node establishes another tunnel to the compute node. 
+In this case, we write (as in the case for Sherlock). The first half is executed on the local machine `ssh -L $PORT:localhost:$PORT ${RESOURCE}`, which establishes a port forwarding to the login node of the HPC. The next line `ssh -L $PORT:localhost:$PORT -N "$MACHINE" &` is run from the login node on the HPC, and port forwards it to the compute node, since you can only access the compute node from the login nodes. The ssh command is:
```

Comment:
```suggestion
In this case (as in the case for Sherlock) we write a command where we port forward to the login node, and then the compute node, which is accessible from the login node. The entire command might look like this:

      $ ssh -L $PORT:localhost:$PORT ${RESOURCE} ssh -L $PORT:localhost:$PORT -N "$MACHINE" &

In the command above, the first half is executed on the local machine `ssh -L $PORT:localhost:$PORT ${RESOURCE}`, which establishes a port forwarding to the login node. The next line `ssh -L $PORT:localhost:$PORT -N "$MACHINE" &` is run from the login node, and port forwards it to the compute node, since you can only access the compute node from the login nodes.
```

Reply:
Did you change it after my edit? I can't tell!